src/sentry/static/sentry/app/views/organizationGroupDetails/groupTagValues.tsx

```diff
@@ -163,13 +164,28 @@ class GroupTagValues extends AsyncComponent<
     );
   }
 }
-const Header = styled('h3')`
+const Header = styled('div')`
   display: flex;
-  align-items: flex-end;
+  align-items: center;
+  margin: 0 0 20px;
+`;
+
+const HeaderTitle = styled('h3')`
+  margin: 0;
+`;
+
+const HeaderButtons = styled('div')`
```

Comment:
You could use `ButtonBar` to do most of this.

Reply:
let me try that